src/app/store/slices/storage/storage.thunks/uploadFolderThunk.ts

```diff
@@ -33,9 +34,11 @@ interface UploadFolderThunkPayload {
   };
 }
 
-const handleFoldersRename = async (root: IRoot, currentFolderId: string, tokenHeader?: string) => {
-  const storageClient = SdkFactory.getNewApiInstance().createNewStorageClient();
-  const [parentFolderContentPromise] = storageClient.getFolderContentByUuid(currentFolderId, false, tokenHeader);
+const handleFoldersRename = async (root: IRoot, currentFolderId: string, workspaceTokenHeader?: string) => {
+  const [parentFolderContentPromise] = newStorageService.getFolderContentByUuid({
+    folderUuid: currentFolderId,
+    workspacesToken: workspaceTokenHeader,
```

Comment:
Remove this change, as in the PR https://github.com/internxt/drive-web/pull/1282 this is removed